### DISCUSSION TITLE
404 on gh-pages

### DISCUSSION THREAD

**Discussion by blehman** [community/community#109694] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Any suggestions the gh-pages site (see [repo](https://github.com/blehman/tweet_bigly)) is throwing a 404? https://blehman.github.io/tweet_bigly/. I have previously used gh-pages successfully to put up a page.

- [working gh-pages site](https://blehman.github.io/boulder_city/)  ([repo](https://github.com/blehman/boulder_city))
- [404 gh-pages site](https://blehman.github.io/tweet_bigly/ ) ([repo](https://github.com/blehman/tweet_bigly))

I've found this [similar question](https://github.com/orgs/community/discussions/108580) and done the troubleshooting [here](https://docs.github.com/en/pages/getting-started-with-github-pages/troubleshooting-404-errors-for-github-pages-sites), but I still get a 404. Any thoughts?

**Comment by namith1003** [comment]
It seems to be functional, maybe you got it fixed I'm guessing.
